README.md

```diff
@@ -67,6 +73,8 @@ To run CLI: `kubectl kudo`
   * [staticcheck](https://github.com/dominikh/go-tools#installation)
   * [kubebuilder](https://book.kubebuilder.io/getting_started/installation_and_setup.html)
 
+Tools are located in `tools.go` to pin their versions. Refer to (https://github.com/go-modules-by-example/index/blob/ac9bf72/010_tools/README.md)[https://github.com/go-modules-by-example/index/blob/ac9bf72/010_tools/README.md] for more information. The Makefile will automatically `go install` the required tools before they are installed, using the versions specified by the module.
```

Comment:
" to pin their versions" -> "in order  to pin their versions"

Reply:
"automatically `go install` the required tools before they are installed" -> "automatically `go install` the required tools before they are needed" . ?

Reply:
"The Makefile will automatically `go install` required tools."?

Reply:
+1